### ISSUE TITLE
autoOpen=true not working on Windows.

### ISSUE THREAD

**Issue by Rup1** [adamgruber/mochawesome#144]
Hi I have the following in my package.json:

```
"scripts": {
    "test": "mocha test --reporter mochawesome --reporter-options autoOpen=true"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "mocha": "^3.2.0",
    "mochawesome": "^2.0.4"
  }
```

When I run 'npm test' I get:

```
PS C:\Users\ConquerJS\Desktop\dev\webdriver-tutorial\webdrivertut> npm test

> webdrivertut@1.0.0 test C:\Users\ConquerJS\Desktop\dev\webdriver-tutorial\webdrivertut
> mocha test --reporter mochawesome --reporter-options autoOpen=true



(node:10312) DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
  library app scenarios
    √ Changes Button opacity upon email being filled out (4015ms)
    √ submitting email shows an alert (3143ms)
    √ Shows a navbar (2292ms)

  3 passing (9s)

[mochawesome] Report JSON saved to C:\Users\ConquerJS\Desktop\dev\webdriver-tutorial\webdrivertut\mochawesome-reports\mochawesome.json

[mochawesome] Report HTML saved to C:\Users\ConquerJS\Desktop\dev\webdriver-tutorial\webdrivertut\mochawesome-reports\mochawesome.html

PS C:\Users\ConquerJS\Desktop\dev\webdriver-tutorial\webdrivertut>

```

But the browser doesn't open. 

Initally I had windows set to node version 7.7.3, but now I set both Windows and mac to 6.7.0 and mac works but not windows.

All code is in my repo here https://github.com/Rup1/webdrivertut

**Comment by adamgruber** [comment]
Hey, thanks for the sample repo. From what I can tell, the call to open the file is being made but then the process exits. On Windows this seems to cause the child process to die before it can finish opening the file. I'll have to rework things a bit to ensure that the open completes before we tell mocha we're done.

**Comment by adamgruber** [comment]
Fixed in `mochawesome-report-generator@1.1.2`. Reinstall mochawesome to get the latest version.

**Comment by Rup1** [comment]
BOOM. It works! Thank you for the quick fix and developing this amazing reporter.
